src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
```

Comment:
Might want to start adding javaDocs early to lesser workload in the coming weeks!